src/tools/rbd/action/Kernel.cc

```diff
@@ -163,6 +163,7 @@ static int parse_unmap_options(char *options)
 
 static int do_kernel_showmapped(Formatter *f)
 {
+#if defined(__linux__)
```

Comment:
@tchaikov 
I'm leaving this in for the moment, because I would like to not change the run-cli tests.
And that expects `showmapped map unmap`  in the help output.
So for met that would make this the most transparent fix.